pkg/schema/load.go

```diff
@@ -20,6 +26,14 @@ func LoadSignalsCSV(r io.Reader) ([]*SignalInfo, error) {
 		return nil, fmt.Errorf("failed to read vspec: %w", err)
 	}
 
+	locRec := make([]string, colLen)
+	locRec[nameCol] = vss.FieldCurrentLocationCoordinates
```

Comment:
We want this to be the VSS name `Vehicle.CurrentLocation.Coordinates` is the JSON name 
```suggestion
	locRec[nameCol] = "Vehicle.CurrentLocation.Coordinates"
```